pkg/server/routes.go

```diff
@@ -20,21 +20,24 @@ func (s *server) routes(
 	// trace all requests with an ID
 	s.router.Use(traceMiddleware)
 
-	// add a request based logger
-	s.router.Use(loggerMiddleware)
-
 	// health check goes directly on the main router to avoid auth
 	healthCheckHandler := handlers.HealthCheckHandler{
 		Config: s.Config,
 	}
 	s.router.HandleFunc("/api/v1/healthcheck", healthCheckHandler.Handle())
 
 	// set up CEDAR client
-	cedarClient := cedar.NewTranslatedClient(s.Config.GetString("CEDAR_API_KEY"))
+	cedarClient := cedar.NewTranslatedClient(
+		s.Config.GetString("CEDAR_API_URL"),
+		s.Config.GetString("CEDAR_API_KEY"),
+	)
 
 	// API base path is versioned
 	api := s.router.PathPrefix("/api/v1").Subrouter()
 
+	// add a request based logger
+	api.Use(loggerMiddleware)
```

Comment:
Not a blocker, but why are we moving this here?

Reply:
The healthcheck logs are really spammy, since the ALB hits the endpoint regularly. If we think those are helpful, we can readd them and provide some filters for the logs.

Reply:
Thanks, that's helpful context!